addons/beehave/debug/global_debugger.gd

```diff
@@ -20,7 +20,7 @@ func _on_debug_message(message: String, data: Array) -> bool:
 
 
 func _set_active_tree(tree_id: int) -> void:
-	var tree: BeehaveTree = _registered_trees.get(tree_id, null)
+	var tree = _registered_trees.get(tree_id, null)
```

Comment:
Not ideal but removing `BeehaveTree` from here excludes this class from initial dependency lookups.